Packs/SecurityScorecard/CONTRIBUTORS.json

```diff
@@ -0,0 +1 @@
+["Mariano Mendez"]
```

Comment:
Feel free to also add yourself also to `pack_metadata.json`:
```json
{
    "githubUser": [
            "hx0ar"
     ]
}
```

That way if anyone proposes any changes to the Pack, it will tag you.